model/src/main/proto/profile.proto

```diff
@@ -17,6 +17,15 @@ message ProfileDatabase {
   map<int32, Profile> profiles = 3;
 }
 
+// Structure for a admin.
+message AdminSettings{
+  // Determines whether profile can download and update content only on wifi.
```

Comment:
`// Determines whether to download and update content only on wifi.`

Reply:
Done.